MovieQuiz/Services/AlertPresenter.swift

```diff
@@ -30,8 +28,9 @@ class AlertPresenter {
         
         alert.addAction(action)
         DispatchQueue.main.async {
-            self.controller.present(alert, animated: true, completion: nil)
+            if let safeController = self.controller {
+                safeController.present(alert, animated: true, completion: nil)
+            }
```

Comment:
Достаточно и так:
```suggestion
            self.controller?.present(alert, animated: true, completion: nil)
```
Распаковка тоже вариант, но здесь он излишний. Так вы в одну строку говорите "выполнить код если ссылка существует, если нет – не выполняй")

Reply:
для отвальных случаев тоже так можно поступить